benchmark/src/main/java/com/linecorp/decaton/benchmark/BenchmarkResult.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.decaton.benchmark;
+
+import java.io.OutputStream;
+import java.io.PrintWriter;
+import java.time.Duration;
+import java.util.Map.Entry;
+
+import lombok.Value;
+import lombok.experimental.Accessors;
+
+@Value
+@Accessors(fluent = true)
+public class BenchmarkResult {
+    @Value
+    public static class Durations {
```

Comment:
`LatencyStats` sounds more natural I think

Reply:
I thought that it's not necessary be "latency" but anything that measures "duration"...